e2e/content-services/permissions-component.e2e.ts

```diff
@@ -245,4 +261,56 @@ describe('Permissions Component', function () {
         });
 
     });
+
+    describe('Role Consumer', function () {
+
+        beforeEach(async (done) => {
+
+            loginPage.loginToContentServicesUsingUserModel(fileOwnerUser);
+            contentServicesPage.goToDocumentList();
+
+            contentList.checkContentIsDisplayed(roleFolder.name);
+            contentServicesPage.checkSelectedSiteIsDisplayed('My files');
+            contentList.clickRowMenuActionsButton(roleFolder.name);
+            contentList.clickMenuActionNamed('PERMISSION');
+            permissionsPage.checkAddPermissionButtonIsDisplayed();
+            permissionsPage.clickAddPermissionButton();
+            permissionsPage.checkAddPermissionDialogIsDisplayed();
+            permissionsPage.checkSearchUserInputIsDisplayed();
+            permissionsPage.searchUserOrGroup(filePermissionUser.getId());
+            permissionsPage.clickUserOrGroup(filePermissionUser.getFirstName());
+            permissionsPage.checkUserOrGroupIsAdded(filePermissionUser.getId());
+            permissionsPage.clickRoleDropdown();
+            permissionsPage.selectOption('Consumer');
+            done();
+        });
+
+        it('[C274691] Should not to be able to Delete, see Info or Upload a file with Consumer permission', () => {
+
+            loginPage.loginToContentServicesUsingUserModel(filePermissionUser);
+            contentServicesPage.goToDocumentList();
+            browser.sleep(15000); // for the Search indexing to finish
```

Comment:
This sleep is not needed, the indexing is done only after creating the folder.

Reply:
Please let me know, if there is a workaround for this.

Reply:
I couldn't find one, it needs more investigation. I'm fine keeping it for now.